src/hashbang.html

```diff
@@ -61,7 +69,7 @@
     <script src='/assets/icon.js'></script>
     <h1>#!</h1>
     <a href="view-source:https://hashbang.sh">
-      <code>curl hashbang.sh | gpg > hashbang.sh</code>
+      <code>curl hashbang.sh | gpg | sh</code>
```

Comment:
I think the issue might be with pumping GPG (or any output containing the command `read`) into sh where sh starts reading. You'd have to do something like `sh <(curl https://hashbang.sh | gpg)` but, again, DON'T do this.